lib/ansible/modules/packaging/os/apt.py

```diff
@@ -124,6 +124,7 @@
    - python-apt (python 2)
    - python3-apt (python 3)
    - aptitude (before 2.4)
+   - xz-utils
```

Comment:
If `xz-utils` is only a requirement with the `deb` parameter, please don't add it here - this section is for general requirements for the module.